cloudformation/infrastructure/jenkins-server.yaml

```diff
@@ -44,10 +44,9 @@ Resources:
                       software-properties-common \
                       docker-ce docker-ce-cli containerd.io \
                       openjdk-8-jdk jenkins \
-                      nginx
+                      nginx awscli
                     usermod -aG docker jenkins
                     snap install postgresql96
-                    pip3 install awscli --upgrade
```

Comment:
This is an interesting judgement call: awscli releases frequently enough that I tend to pip install it to get access to new features but if we're not using it having that be cached should be more robust.